cameo/flux_analysis/analysis.py

```diff
@@ -506,7 +506,10 @@ def _get_c_source_reaction(model):
     source_reactions = [(reaction, reaction.flux * n_carbon(reaction)) for reaction in model_reactions if
```

Comment:
Multiply by the stoichiometric coefficient as well? It **should** be 1 but you never know what people do with their models :wink: Or is that already part of `n_carbon`?